uportal-war/src/main/java/org/apereo/portal/shell/PortalShell.java

```diff
@@ -76,9 +76,6 @@ public static void main(String[] args) throws Exception {
                         new CompilerConfiguration(System.getProperties());
                 final GroovyShell shell = new GroovyShell(binding, conf);
                 shell.run(scriptFile, args);
-            } else {
```

Comment:
A parameter less version of `Groovysh.run()` no longer exists.

reference: http://docs.groovy-lang.org/latest/html/api/groovy/lang/GroovyShell.html